### DISCUSSION TITLE
Documentation about how total value is calculated

### DISCUSSION THREAD

**Discussion by rivaser** [boyter/scc#383] [Q&A] [answered]
Hi,
I am trying to find about how SCC is calculating the Estimated Cost to Develop
Please can you help me out?
Thanks

**Comment by boyter** [comment, marked as answer]
Its done through the COCOMO model. You can find some details here https://github.com/boyter/scc#cocomo with the implementation being here https://github.com/boyter/scc/blob/master/processor/cocomo.go

The wikipedia entry is worth looking at as well https://en.wikipedia.org/wiki/COCOMO and covers some of the ideas.

`scc` is setup to mimic `sloccount` https://dwheeler.com/sloccount/ and its calculation where the average wage per year is set to 56286 (you can override this using the `--avg-wage` command line option). The COCOMO project type by default is set to `organic` but you can change this to one of the other types or supply a totally custom calculation if you chose.

If you have a specific question about it please let me know and I will attempt to help.

**Reply by rivaser** [reply]
@boyter thanks a lot for answering my question! 👍

**Reply by boyter** [reply]
@rivaser you are welcome. 

It might look like `scc` is unmaintained, but the reality is I have just been busy on other things. I have some things in progress now that should allow me to get it moving again.
